src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,76 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './Loader.module.css';
+
+// TODO: some props are shadowed from react-loader-spinner. Improve?
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
+
+  /**
+   * Aria label of the oval. Default is oval-loading. Will be overridden if ariaLabel is passed in props
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
+   */
+  ariaLabel?: string;
+
+  /**
+   * Whether the oval is visible. Default is true.
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
+   */
+  visible?: boolean;
+};
+
+// Pixel sizes corresponding to EDS size units (pixel = rem * 16)
+const loaderSize = {
+  sm: 16, // --eds-size-2
+  md: 40, // --eds-size-5
+  lg: 80, // --eds-size-10
+};
+
+// Given a loader size, the stroke widths can change
+const loaderStrokeSize = {
+  sm: 1,
+  md: 2,
+  lg: 3,
+};
+
+/**
+ * BETA: This component is still a work in progress and is subject to change.
+ *
+ * `import {Loader} from "@chanzuckerberg/eds";`
+ *
+ */
```

Comment:
Think we generally include a short blurb about the component here in the docstring

Reply:
Yah forgot to add something as a little starter. We will have some revisions to this once Sean gets a design up for comparison.